### DISCUSSION TITLE
Considering going for RC.1

### DISCUSSION THREAD

**Discussion by pauldendulk** [Mapsui/Mapsui#1955] [Development]
There were a couple of things I wanted to do before the release but it looks like it will take too much time to complete, while there are new platforms, which have no v3 version, that waiting to be used.

The things I had planned before the v3 release:
- Rewrite Viewport/Navigator. This is done and I am happy with it.
- Make MVVM easier. This is done in part and I am not sure what else is really needed. The Navigator and the Refresh methods are now moved to the Map which makes it possible for most scenarios to use only the Map in the ViewModel, but I assume there is more work to be done here, but that could be done in a next RC or a 4.1 release.
- Rendering rewrite: The render loop does nothing but draw skia object (which are built in a previous stage) to the canvas. I did nothing on this.

The rendering would be part of v5. Also I did not get into many issues regarding rendering during the last month so that makes it feel less urgent to me.

The rewriting of the Viewport/Navigator did trigger some changes to the MapControl/MapView and this is something I would also like to improve, but this would postpone the v4 release even more.

I would like to hear what you think, both developers and users. Particularly @inforithmics who is responsible for most of the new platforms.

**Comment by inforithmics** [comment]
For me this is fine. The platforms seem to be more or less stable, except for UNO because there we are waiting for a fix for XamlRoot.RasterizationScale.

When this fix is avaible I plan to raise the minimum Uno Requirment, and eventually use the the new Uno Template with a multitarget project instead of a shared project.

**Comment by pauldendulk** [comment]
With the RasterizationScale workaround it behaves correctly (it centers correctly) only the resolution is not fully correct. We may be able to upgrade before the v4 release.

>  I plan to raise the minimum Uno Requirment, and eventually use the the new Uno Template with a multitarget project instead of a shared project.

That would be a nice improvement but would only affect the samples, no?

**Reply by inforithmics** [reply]
Yes, so the nugets shouldn't change.
